doc/Command-Reference.md

```diff
@@ -10364,44 +10364,85 @@ This command is used to install a new image on the alternate image partition.  T
   ```
 
 - Example:
-  ```
-  admin@sonic:~$ sudo sonic-installer install https://sonic-jenkins.westus.cloudapp.azure.com/job/xxxx/job/buildimage-xxxx-all/xxx/artifact/target/sonic-xxxx.bin
-  New image will be installed, continue? [y/N]: y
-  Downloading image...
-  ...100%, 480 MB, 3357 KB/s, 146 seconds passed
-  Command: /tmp/sonic_image
-  Verifying image checksum ... OK.
-  Preparing image archive ... OK.
-  ONIE Installer: platform: XXXX
-  onie_platform:
-  Installing SONiC in SONiC
-  Installing SONiC to /host/image-xxxx
-  Directory /host/image-xxxx/ already exists. Cleaning up...
-  Archive:  fs.zip
-     creating: /host/image-xxxx/boot/
-    inflating: /host/image-xxxx/boot/vmlinuz-3.16.0-4-amd64
-    inflating: /host/image-xxxx/boot/config-3.16.0-4-amd64
-    inflating: /host/image-xxxx/boot/System.map-3.16.0-4-amd64
-    inflating: /host/image-xxxx/boot/initrd.img-3.16.0-4-amd64
-     creating: /host/image-xxxx/platform/
-   extracting: /host/image-xxxx/platform/firsttime
-    inflating: /host/image-xxxx/fs.squashfs
-    inflating: /host/image-xxxx/dockerfs.tar.gz
-  Log file system already exists. Size: 4096MB
-  Installed SONiC base image SONiC-OS successfully
-
-  Command: cp /etc/sonic/minigraph.xml /host/
 
-  Command: grub-set-default --boot-directory=/host 0
-
-  Done
-  ```
+```bash
```

Comment:
Need add 2 spaces to align with `- Example:`
![image](https://user-images.githubusercontent.com/6898708/196605351-a3133495-6f29-4c49-9117-3d743a3d258a.png)

Reply:
Thanks for the suggestion! Done.